packages/python-sdk/e2b/sandbox_sync/main.py

```diff
@@ -244,12 +262,17 @@ def connect(
         same_sandbox = Sandbox.connect(sandbox_id)
         ```
         """
+
+        response = SandboxApi.get_info(sandbox_id)
```

Comment:
i think you want to do that in __init__ (it's okay in Async, as Async can't be called only with __init__

Reply:
Done, also cleans up envd params from init method.